### PR TITLE
Development alex

### PR DESCRIPTION
**Pull request by pytzcarraldo**
Tests ran successfully local.

### CODE REVIEW

File: gempy/posterior_analysis.py [hunk 1]
```diff
@@ -131,7 +145,7 @@ def compute_probability_lithology(lith_blocks):
     return lith_prob
 
 
-def calcualte_information_entropy(lith_prob):
+def calculate_information_entropy(lith_prob):
```
Comment (Leguark) [inline gempy/posterior_analysis.py]: I liked it more before

File: gempy/posterior_analysis.py [hunk 2]
```diff
@@ -222,7 +246,7 @@ def change_input_data(self, interp_data, i):
         # replace interface data
         interp_data.geo_data_res.interfaces[["X", "Y", "Z"]] = self.input_data[i][0]
         # replace foliation data
-        interp_data.geo_data_res.orientations[["G_x", "G_y", "G_z", "X", "Y", "Z", "dip", "azimuth", "polarity"]] = self.input_data[i][1]
+        interp_data.geo_data_res.orientations[["X", "Y", "Z", "dip", "azimuth", "polarity"]] = self.input_data[i][1]
```
Comment (Leguark) [inline gempy/posterior_analysis.py]: Did you delete this on purpose or is one a mismatch between your code and mine?

File: gempy/posterior_analysis.py [hunk 3]
```diff
@@ -232,38 +256,27 @@ def change_input_data(self, interp_data, i):
...
 
+        # average input data
+        interf_avrg = self.input_data[:, 0][:].mean(axis=0)
+        orient_avrg = self.input_data[:, 1][:].mean(axis=0)
+        # set average in df
         interp_data.geo_data_res.interfaces[["X", "Y", "Z"]] = interf_avrg
-        interp_data.geo_data_res.orientations[["G_x", "G_y", "G_z", "X", "Y", "Z", "dip", "azimuth", "polarity"]] = fol_avrg
-        interp_data.update_interpolator()
-        return gp.compute_model(interp_data)
+        interp_data.geo_data_res.orientations[["X", "Y", "Z", "dip", "azimuth", "polarity"]] = orient_avrg
```
Comment (Leguark) [inline gempy/posterior_analysis.py]: Same here

File: gempy/posterior_analysis.py (hunk 1 above)
Comment (pytzcarraldo) [reply gempy/posterior_analysis.py]: agreed

File: gempy/posterior_analysis.py (hunk 2 above)
Comment (pytzcarraldo) [reply gempy/posterior_analysis.py]: deleted on purpose as in legacy script I never saved gradient values, class will become deprecated in future anyways

File: gempy/posterior_analysis.py (hunk 3 above)
Comment (pytzcarraldo) [reply gempy/posterior_analysis.py]: same here

**Pull request merged**